Submitted By: Jim Gifford <jim at linuxfromscratch dot org>
Date: 2006-07-04
Initial Package Version: 7.15.4
Origin: Various
Description: Make curl-onfig Multilib Friendly

diff -Naur curl-7.15.4.orig/curl-config.in curl-7.15.4/curl-config.in
--- curl-7.15.4.orig/curl-config.in	2006-05-02 15:48:22.000000000 -0700
+++ curl-7.15.4/curl-config.in	2006-07-04 12:42:51.425803497 -0700
@@ -181,7 +181,7 @@
        	;;
 
     --libs)
-       	echo -L@libdir@ -lcurl @LDFLAGS@ @LIBS@
+       	echo -lcurl @LDFLAGS@ @LIBS@
        	;;
 
     *)
